examples/platforms/cc2538/random.c

```diff
@@ -107,15 +109,15 @@ ThreadError otPlatRandomSecureGet(uint16_t aInputLength, uint8_t *aOutput, uint1
     {
         channel = 11 + (HWREG(RFCORE_XREG_FREQCTRL) - 11) / 5;
         otPlatRadioSleep(sInstance);
-        otPlatRadioDisable(sInstance);
+        sState = kStateDisabled;
```

Comment:
I believe these lines could just be removed, as setting the state (what it essentially did before) doesn't matter. But in looking at this, I think there is a bug here related to calling `otPlatRadioSleep`. It won't disable the receiver if it is in the transmit state. I don't know what the effect of trying to do all this while in that state, but I am guessing at the very least the transmit will fail, worst case everything gets out of sync and causes a crash.